clients/cloud/python/README.md

```diff
@@ -69,3 +69,94 @@ Consumed record with key alice and value {"count": 9}, and updated total count t
 Waiting for message or event/error in poll()
 ...
 ```
+
+
+# Example 2: Avro And Confluent Cloud Schema Registry
```

Comment:
Can we add a note about the regression introduced when not specifying the root CA.

Reply:
Do you mean https://github.com/confluentinc/confluent-kafka-python/pull/578 ?
Reasons to NOT mention it:

1. It results in a warning and does NOT impact functionality
2. Slippery slope: this isn't the place to mention regressions/bugs/etc

Reply:
Reasons to mention it: 
1. WARN messages scare new users in the first 15 
2. It is not the place for all bugs no but it is the place to people who are unfamiliar with our clients and possibly python. These types of errors throw new comers off and can send them  searching for an issue that may not be there for hours. This arguable makes for a terrible first 15. 

Anyway I'll leave it to your discretion.

Reply:
Compromise: I added the warning message to confluentinc/confluent-kafka-python#578 so at least it's searchable.